tests/e2e/example.spec.js

```diff
@@ -16,7 +16,7 @@ describe('example test', () => {
         await visitAdminPage('plugins.php');
 
         // Select the plugin based on slug and active class
-        const activePlugin = await page.$x('//tr[contains(@class, "active") and contains(@data-slug, "wpgraphql-content-blocks")]');
+        const activePlugin = await page.$x('//tr[contains(@class, "active") and not(contains(@class, "plugin-update-tr")) and contains(@data-slug, "wpgraphql-content-blocks")]');
```

Comment:
I added this because the plugin updater code adds an 'active' row to display any update-related errors. See the commit message here: https://github.com/wpengine/wp-graphql-content-blocks/pull/179/commits/db0c0810f641e4c3dfa7e8f03082bd6b914f12b2

Note: at the time of this comment, this test file fails before this runs, at the user login stage.
Screenshot:
![image](https://github.com/wpengine/wp-graphql-content-blocks/assets/1254870/b6abf41a-90e6-4855-a31f-faa3e77eb28d)

Link to failure: https://github.com/wpengine/wp-graphql-content-blocks/actions/runs/7786743891/job/21232252228

These tests pass locally for me using `npm run wp-env start` and `npm run test-e2e`.